### PR TITLE
Docs: Updating Vue 3 readme

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR:
- Updates the Vue 3 readme file (fixing #9009)
- Fixes typos in the other wrappers' readme files
- Updates documentation links in the other wrappers' readme files, to limit redirects
- Adds a `vue 3` keyword to the Vue 3 wrapper's package.json
- Adds a Vue 3 icon to the main Handsontable README files (both `./README.md` and `./handsontable/README.md`)

[skip changelog]

### CODE REVIEW

**Review by budnix — COMMENTED** [review]
As you change all project-related READMEs maybe it's a good idea to update the URL to new ones? For example:
 * https://handsontable.com/docs/Core.html -> https://handsontable.com/docs/api/core/
 * https://handsontable.com/docs/tutorial-release-notes.html -> https://handsontable.com/docs/release-notes/
 * https://handsontable.com/examples -> https://handsontable.com/demo
 * https://handsontable.com/docs/tutorial-license-key.html -> https://handsontable.com/docs/license-key/

From a user perspective, fewer redirects mean faster the page loads :)

File: wrappers/vue3/README.md [hunk 1]
```diff
@@ -74,34 +74,41 @@ Use this data grid as you would any other component in your application. [Option
...
     }
-  }
+  });
 </script>
+
+<style src="handsontable/dist/handsontable.full.css"></style>
 ```
 
-### [View live demo](//handsontable.com/docs/frameworks-wrapper-for-vue-simple-example.html)
+### [View live demo](//handsontable.com/docs/vue3-simple-example.html)
```
Comment (budnix) [inline wrappers/vue3/README.md:111]:
```suggestion
### [View live demo](https://handsontable.com/docs/next/vue3-simple-example/)
```

**Comment by kirszenbaum** [comment]
> As you change all project-related READMEs maybe it's a good idea to update the URL to new ones? For example:
> 
> * https://handsontable.com/docs/Core.html -> https://handsontable.com/docs/api/core/
> * https://handsontable.com/docs/tutorial-release-notes.html -> https://handsontable.com/docs/release-notes/
> * https://handsontable.com/examples -> https://handsontable.com/demo
> * https://handsontable.com/docs/tutorial-license-key.html -> https://handsontable.com/docs/license-key/
> 
> From a user perspective, fewer redirects mean faster the page loads :)

@budnix Sure, let's do this: e03ef5a

File: wrappers/vue3/README.md (hunk 1 above)
Comment (kirszenbaum) [reply wrappers/vue3/README.md]: @budnix - I updated all such links here: e03ef5a

**Review by budnix — COMMENTED** [review]
Can we add Vue 3 wrapper to the "Get Started with Handsontable" section of the main `./README.md` file?
image

**Comment by kirszenbaum** [comment]
> Can we add Vue 3 wrapper to the "Get Started with Handsontable" section of the main `./README.md` file? image

@budnix - sure, done: 7db6e9a

File: wrappers/react/README.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ class HotApp extends React.Component {
 }
 ```
 
-### [View live demo](//handsontable.com/docs/frameworks-wrapper-for-react-simple-examples.html)
+### [View live demo](https:handsontable.com/docs/frameworks-wrapper-for-react-simple-examples.html)
```
Comment (budnix) [inline wrappers/react/README.md:100]:
```suggestion
### [View live demo](https://handsontable.com/docs/frameworks-wrapper-for-react-simple-examples.html)
```

File: wrappers/vue/README.md [hunk 3]
```diff
@@ -101,7 +101,7 @@ Use this data grid as you would any other component in your application. [Option
 </script>
 ```
 
-### [View live demo](//handsontable.com/docs/frameworks-wrapper-for-vue-simple-example.html)
+### [View live demo](https:handsontable.com/docs/frameworks-wrapper-for-vue-simple-example.html)
```
Comment (budnix) [inline wrappers/vue/README.md:104]:
```suggestion
### [View live demo](https://handsontable.com/docs/frameworks-wrapper-for-vue-simple-example.html)
```

File: wrappers/vue3/README.md [hunk 4]
```diff
@@ -67,41 +67,48 @@ The component will be available as `Handsontable.vue.HotTable`.
 
 ### Usage
 
-Use this data grid as you would any other component in your application. [Options](//handsontable.com/docs/Options.html) can be set as `HotTable` props.
+Use this data grid as you would any other component in your application. [Options](https:handsontable.com/docs/Options.html) can be set as `HotTable` props.
```
Comment (budnix) [inline wrappers/vue3/README.md:70]:
```suggestion
Use this data grid as you would any other component in your application. [Options](https://handsontable.com/docs/api/options/) can be set as `HotTable` props.
```

File: wrappers/vue/README.md [hunk 5]
```diff
@@ -67,7 +67,7 @@ The component will be available as `Handsontable.vue.HotTable`.
 
 ### Usage
 
-Use this data grid as you would any other component in your application. [Options](//handsontable.com/docs/Options.html) can be set as `HotTable` props.
+Use this data grid as you would any other component in your application. [Options](https:handsontable.com/docs/Options.html) can be set as `HotTable` props.
```
Comment (budnix) [inline wrappers/vue/README.md]:
```suggestion
Use this data grid as you would any other component in your application. [Options](https://handsontable.com/docs/api/options/) can be set as `HotTable` props.
```

File: wrappers/react/README.md (hunk 2 above)
Comment (kirszenbaum) [reply wrappers/react/README.md]: @budnix Oops, fixed: ee72d61

File: wrappers/vue/README.md (hunk 3 above)
Comment (kirszenbaum) [reply wrappers/vue/README.md]: @budnix Oops, fixed: ee72d61

File: wrappers/vue3/README.md (hunk 4 above)
Comment (kirszenbaum) [reply wrappers/vue3/README.md]: @budnix Oops, fixed: ee72d61

File: wrappers/vue/README.md (hunk 5 above)
Comment (kirszenbaum) [reply wrappers/vue/README.md]: @budnix - done for all `Options.html` links: 67a6f3b

File: wrappers/react/README.md [hunk 6]
```diff
@@ -97,7 +97,7 @@ class HotApp extends React.Component {
 }
 ```
 
-### [View live demo](//handsontable.com/docs/frameworks-wrapper-for-react-simple-examples.html)
+### [View live demo](https://handsontable.com/docs/frameworks-wrapper-for-react-simple-examples.html)
```
Comment (budnix) [inline wrappers/react/README.md:100]:
```suggestion
### [View live demo](https://handsontable.com/docs/react-simple-example/)
```

File: wrappers/vue/README.md [hunk 7]
```diff
@@ -101,7 +101,7 @@ Use this data grid as you would any other component in your application. [Option
 </script>
 ```
 
-### [View live demo](//handsontable.com/docs/frameworks-wrapper-for-vue-simple-example.html)
+### [View live demo](https://handsontable.com/docs/frameworks-wrapper-for-vue-simple-example.html)
```
Comment (budnix) [inline wrappers/vue/README.md:104]:
```suggestion
### [View live demo](https://handsontable.com/docs/vue-simple-example/)
```

File: wrappers/vue3/README.md [hunk 8]
```diff
@@ -67,41 +67,48 @@ The component will be available as `Handsontable.vue.HotTable`.
...
     }
-  }
+  });
 </script>
+
+<style src="handsontable/dist/handsontable.full.css"></style>
 ```
 
-### [View live demo](//handsontable.com/docs/frameworks-wrapper-for-vue-simple-example.html)
+### [View live demo](https://handsontable.com/docs/vue3-simple-example.html)
```
Comment (budnix) [inline wrappers/vue3/README.md:111]:
```suggestion
### [View live demo](https://handsontable.com/docs/vue3-simple-example/)
```

Comment (kirszenbaum) [reply wrappers/vue3/README.md]: @budnix - done: e52b630

File: wrappers/vue/README.md (hunk 7 above)
Comment (kirszenbaum) [reply wrappers/vue/README.md]: @budnix - done: e52b630

**Review by budnix — DISMISSED** [review]
Thanks 🥇

**Review by budnix — APPROVED** [review]

**Pull request merged**